api/handlers_api.go

```diff
@@ -161,49 +166,50 @@ func (w *Request) HandleAttack() (*md.Message[md.RespAttack], *md.Player) {
 	// Check what is in the position of attack in defence grid matrix of defender
 	positionCode, err := defender.FetchDefenceGridPositionCode(x, y)
 	if err != nil {
+		// Invalid position in defender defence grid (already hit)
 		resp.AddError(err.Error(), cerr.ConstErrAttack)
 		return &resp, defender
 	}
 
+	// Change the status of players turn
 	defer func() {
-		// Change the status of players turn
 		attacker.IsTurn = false
 		defender.IsTurn = true
 	}()
 
 	// If the attacker missed
-	if positionCode == md.PositionStateDefenceGridEmpty {
+	if positionCode == md.PositionStateAttackGridMiss {
 		resp.AddPayload(md.RespAttack{
-			PositionState: md.PositionStateAttackGridMiss,
-			IsTurn:        attacker.IsTurn,
+			X:               x,
+			Y:               y,
+			PositionState:   md.PositionStateAttackGridMiss,
+			SunkenShipsHost: game.HostPlayer.SunkenShips,
+			SunkenShipsJoin: game.JoinPlayer.SunkenShips,
```

Comment:
Am I blind or you forgot to pass `IsTurn` in the `RespAttack` ?

Reply:
Nvm I saw what you did there

Reply:
Lool, yea I put it down there to reflect the final status of sunkenships